### DISCUSSION TITLE
Github error 404

### DISCUSSION THREAD

**Discussion by ammar1999-a** [community/community#23645] [Other Feature Feedback, Questions, & Ideas]
<p>I made my repository and I got GitHub like of my site but whenever I try to go to the site i get error 404 says the file not found any help plz</p>

**Comment by kingthorin** [comment]
<p>Links? Hard to help without seeing what you’re talking about.</p>

**Comment by ammar1999-a** [comment]
<p>This is the link of my repo <a href="https://ammar1999-a.github.io/" rel="noopener nofollow ugc">https://ammar1999-a.github.io/</a> but it just shows the name not the site so can u help plz</p>

**Comment by airtower-luna** [comment]
<p>Looks like <a href="https://github.com/ammar1999-a/ammar1999-a.github.io">https://github.com/ammar1999-a/ammar1999-a.github.io</a> is the repository. The repository doesn’t have an index.html file, so by default the README is rendered.</p>
<p>I assume <a href="https://ammar1999-a.github.io/Accounting.html">https://ammar1999-a.github.io/Accounting.html</a> is what you want?</p>

**Comment by ammar1999-a** [comment]
<p>Ok that’s right now I can see my site but what did do exactly and I also uploaded  a css and JavaScript files in the repo why I can’t see them ? Just the html file is viewed</p>

**Comment by ammar1999-a** [comment]
<p>I found out that when I write.HTML. It views the html doc and when I write.CSS it views the css doc but I want to view my whole project with HTML , CSS and JAVASCRIPT</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>The paths with which the HTML file references the CSS and script files are wrong. Here’s what they look like in your HTML file:</p>
<pre><code class="lang-auto">&lt;link rel="stylesheet" href="css/Accounting.css"&gt;
&lt;script src="js/accounting.js"&gt;&lt;/script&gt;
</code></pre>
<p>However, looking at the repository the files are in the same directory as the HTML file, not in <code>css/</code> and <code>js/</code> subdirectories. So the references should be:</p>
<pre><code class="lang-auto">&lt;link rel="stylesheet" href="Accounting.css"&gt;
&lt;script src="accounting.js"&gt;&lt;/script&gt;
</code></pre>

**Reply by IssahYaakub** [reply]
i am facing the same erro please help me out.

**Reply by Singhamit1995** [reply]
https://github.com/Singhamit1995/Indian-General-Election-2024 is not opens showing 404 not found

**Reply by Phoenix399** [reply]
mine is my codespace please i need help

**Comment by ammar1999-a** [comment]
<p>Thank you so much now it’s working perfectly</p>
